pkg/controllers/coscheduling/podgroup_controller.go

```diff
@@ -47,16 +66,158 @@ type PodGroupReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.11.0/pkg/reconcile
 func (r *PodGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	log := log.FromContext(ctx)
+	pg := &schedv1alpha1.PodGroup{}
+	if err := r.Get(ctx, req.NamespacedName, pg); err != nil {
+		if apierrs.IsNotFound(err) {
+			log.V(5).Info("Pod group has been deleted")
```

Comment:
Need a print namespace and name of this PodGroup

Reply:
the logger from controller-runtime context is printing namespace and name by default

Reply:
OK